docs/octopus-cloud/data.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Data
+position: 10
+description: How to work with your data in Octopus Cloud.
+---
+
+We make regular backups of instance databases throughout the day and store these securely in blob storage, so we have multiple restore points in the unlikely case of disaster.
```

Comment:
I wonder if we ought to call out how the data is secured/encrypted here (or even just that "it is"). @pawelpabich?

Reply:
> We make regular backups of instance databases throughout the day 
Not sure this is 100% correct. I think we only do it when we make changes to the customer's instance. @stopthatastronaut  would now more.

Reply:
> in blob storage
Do you think we need to go to his level of detail ? Wouldn't `store these securely.` be enough?

Reply:
> how the data is secured/encrypted here 
Personally I would avoid describing **how** we implemented Hosted as this will get out sync quickly and I'm not sure customers will get a lot of benefits from that knowledge.

Reply:
Updated to:
We make backups of instance databases and store these securely, so we have multiple restore points in the unlikely event of a disaster.